### PR TITLE
feat(launch): Add prior runs when creating a sweep from the CLI

### PR DESCRIPTION
**Pull request by TimH98**
Description
-----------

- Fixes [WB-19251](https://wandb.atlassian.net/browse/WB-19251)

We already have the ability to add already-completed runs to a sweep from the UI. This PR adds the ability to do the same using the command line, using `wandb sweep -R <run id> -R <run id> ...`

- [x] I updated CHANGELOG.md, or it's not applicable

Testing
-------

A unit test was added.

Tested manually using the following: `wandb sweep -R cq35n9a1 -R oobmkxxw sweep_config.yaml -e tim-hays -p debugging`, and confirming in the UI that the sweep had the two runs added to it (the contents of the runs and the sweep config are not important).

[WB-19251]: https://wandb.atlassian.net/browse/WB-19251?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dmitryduev — APPROVED** [review]

File: wandb/sdk/internal/internal_api.py [hunk 1]
```diff
@@ -3070,7 +3072,8 @@ def upsert_sweep(
             $projectName: String,
             $controller: JSONString,
             $scheduler: JSONString,
-            $state: String
+            $state: String,
+            $priorRunsFilters: JSONString,
```
Comment (dmitryduev) [inline wandb/sdk/internal/internal_api.py:3076]: has this field always been there?

Comment (KyleGoyette) [reply wandb/sdk/internal/internal_api.py:3076]: ~ 4 years

File: wandb/cli/cli.py [hunk 2]
```diff
@@ -1104,6 +1114,14 @@ def _parse_settings(settings):
     default=None,
     help="Resume a launch sweep by passing an 8-char sweep id. Queue required",
 )
+@click.option(
+    "--prior_run",
+    "-R",
+    "prior_runs",
+    multiple=True,
+    default=None,
+    help="ID of an existing run to add to this sweep",
```
Comment (KyleGoyette) [inline wandb/cli/cli.py:1123]: I'm not sure users know what a run id is. I don't want to make the help here too convoluted, but would it help users to note that it is the `public.Run.id` attribute?

Comment (KyleGoyette) [reply wandb/cli/cli.py:1123]: Run id is referred to on the run documentation. Should be fine.

**Review by KyleGoyette — APPROVED** [review]
LGTM!

**Pull request merged**